public/docs/ts/latest/guide/server-communication.jade

```diff
@@ -14,34 +14,49 @@ include ../_util-fns
   
   The Angular HTTP client library simplifies application programming of the **XHR** and **JSONP** APIs
   as we'll learn in this chapter covering:
-  
-  [Http client sample overview](#http-client)<br>
-  [Fetch data with http.get](#fetch-data)<br>
-  [RxJS Observable of HTTP Responses](#rxjs)<br>
-  [Enabling RxJS Operators](#enable-rxjs-operators)<br>
-  [Extract JSON data with RxJS map](#map)<br>
-  [Error handling](#error-handling)<br>
-  [Send data to the server](#update)<br>
-  [Add headers](#headers)<br>
-  [Promises instead of observables](#promises)<br>
-  [JSONP](#jsonp)<br>
-  [Set query string parameters](#search-parameters)<br>
-  [Debounce search term input](#more-observables)<br>
-  [Appendix: the in-memory web api service](#in-mem-web-api)<br>
-  
-  We illustrate these topics with code that you can 
-  [run live in a browser](/resources/live-examples/server-communication/ts/plnkr.html).
 
+ul
```

Comment:
any good reason for changing the old convention to this one? Seems more complicated for me.